server/accounts.go

```diff
@@ -2936,17 +2941,33 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 			incompleteImports = append(incompleteImports, i)
 			continue
 		}
+		from := string(i.Subject)
```

Comment:
This not already a string?

Reply:
no, it's coming from the jwt library where we have a subject type